lib/welcome/pages/welcome_page.dart

```diff
@@ -65,6 +67,17 @@ class _WelcomePageState extends State<WelcomePage> {
       child: Column(
         mainAxisAlignment: MainAxisAlignment.center,
         children: [
+          Hero(
```

Comment:
Dieses Hero funktioniert noch nicht oder?

Reply:
Da müsstest du ein Hero mit demselben Tag um den Search-Button in der SearchRidePage machen

Reply:
Bitte auch nochmal diesen Teil der WelcomePage anschauen:
`child: height > 300
                    ? Column(
                        children: [
                          SizedBox(height: height / 2, child: carousel),
                          Expanded(child: buttons),
                        ],
                      )
                    : buttons,`
Da schaue ich sozusagen ob das Bild auf die Seite passt. Da die zwei Buttons von vorher ca. 150 Pixel hoch sind und wir die Seite in der Mitte aufteilen wenn das Bild angezeigt wird, zeige ich das Bild nicht mehr an wenn weniger als 300 Pixel verfügbar sind (du kannst das z.B. mit dem Split Mode testen, indem du das Fenster so klein machst, dass es nur noch 1/4 der Höhe des Gerätes ist). Diesen Wert müsstest du mit 3 Buttons anpassen!

Reply:
Hero hat sich geklärt. Aber das mit dem Bild hab ich noch nicht ganz verstanden. Bei mir sehe ich es immer und das mit dem Split Mode bekomm ich bei mir nicht hin also kann ich das dort schlecht prüfen

Reply:
Naja es gibt ja kleinere Handys und andere Wege die App zu benutzen. Split Screen kannst du probieren, indem du in die Recent Apps-Ansicht kommst (mit Gestensteuerung die Bar hochziehen und halten) und dann auf das App-Symbol oben klickst, "Split Screen" wählst und eine App für die andere Seite wählst. Du kannst dann die Bar in der Mitte nach oben und unten ziehen, um das Größenverhältnis der angezeigten Apps auf 3/4 und 1/4 zu ändern. Spätestens wenn die App dann nur noch 1/4 der ursprünglichen Höhe ist, würde man schon auf meinem relativ großen emulierten Pixel 6 nicht mehr alles sehen. Wenn du Pech hast, gibt es jetzt bei dir schon bei 1/2 - 1/2 einen Fehler

Reply:
done